ci/do_ci.sh

```diff
@@ -10,18 +10,19 @@ if [[ "$1" == "bazel.debug" ]]; then
   ln -sf /thirdparty prebuilt
   ln -sf /thirdparty_build prebuilt
   # Not sandboxing, since non-privileged Docker can't do nested namespaces.
-  echo "Building..."
   export CC=gcc-4.9
-  export CXX=g++-4.9
```

Comment:
And not necessary in this change, but we might consider doing a comment pass on some of the main bazel files just to make it easier for new folks (like me).